packages/salesforcedx-apex-replay-debugger/src/core/heapDumpService.ts

```diff
@@ -327,7 +337,8 @@ export class HeapDumpService {
   //   1: '2'
   //   2: '3'
   private createStringFromVarContainer(
-    varContainer: ApexVariableContainer
+    varContainer: ApexVariableContainer,
+    visitedMap: Map<string, null>
```

Comment:
Is there a reason why we use it as a `Map<string, null>`? I think we don't ever use the key so whatever it is fine. But, in that case, why not use a [`Set`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set) since you are just checking for membership?

Reply:
Habit and slight ignorance. Both Map and Set are close to O(1) in lookup times and I keep seeing Set and thinking **Array**

Reply:
Cool, please change it then. Using a `Set`is much clearer rather than a `Map<string, null>` since there is nothing keyed off the value.

Reply:
It's changed with the latest commit.